architecture/networking/routes.adoc

```diff
@@ -602,7 +602,7 @@ For all the items outlined in this section, you can set annotations on the
 |===
 |Variable | Description | Environment Variable Used as Default
 |`*haproxy.router.openshift.io/balance*`| Sets the load-balancing algorithm. Available options are `source`, `roundrobin`, and `leastconn`. | `ROUTER_TCP_BALANCE_SCHEME` for passthrough routes. Otherwise, use `ROUTER_LOAD_BALANCE_ALGORITHM`.
-|`*haproxy.router.openshift.io/disable_cookies*`| Disables the use of cookies to track related connections. If set to `true` or `TRUE`, the balance algorithm is used to choose which back-end serves connections for each incoming HTTP request. |
+|`*router.openshift.io/cookie_name*`| Disables the use of cookies to track related connections. If set to `true` or `TRUE`, the balance algorithm is used to choose which back-end serves connections for each incoming HTTP request. |
 |`*haproxy.router.openshift.io/cookie_name*`| Specifies an optional cookie to be used for
```

Comment:
We should change this line `haproxy.router.openshift.io/cookie_name` to `router.openshift.io/cookie_name` but not the above one. I think the description in the bug may be confusing.